### PR TITLE
Overhaul SDK Tests

### PR DESCRIPTION
**Pull request by abaldwin-Bandwidth**

### CODE REVIEW

File: src/test/java/com/bandwidth/MfaApiTests.java [hunk 1]
```diff
@@ -36,7 +37,9 @@ public void testMfaMessaging() throws Exception {
                 .message("Your temporary {NAME} {SCOPE} code is {CODE}")
                 .build();
 
-        controller.createMessagingTwoFactor(ACCOUNT_ID, body);
+        TwoFactorMessagingResponse response = controller.createMessagingTwoFactor(ACCOUNT_ID, body).getResult();
+        assertNotNull("MessageID is null", response.getMessageId());
+        assertFalse("MessageID is empty", response.getMessageId().isEmpty());
```
Comment (ajrice6713) [inline src/test/java/com/bandwidth/MfaApiTests.java:42]: I think we want to check for status codes in the response body as well as the length of the message ID string to ensure it isnt changing (that has broken customers previously)

File: src/test/java/com/bandwidth/MfaApiTests.java [hunk 2]
```diff
@@ -50,7 +53,9 @@ public void testMfaVoice() throws Exception {
                 .message("Your temporary {NAME} {SCOPE} code is {CODE}")
                 .build();
 
-        controller.createVoiceTwoFactor(ACCOUNT_ID, body);
+        TwoFactorVoiceResponse response = controller.createVoiceTwoFactor(ACCOUNT_ID, body).getResult();
+        assertNotNull("CallID is null", response.getCallId());
+        assertFalse("CallID is empty", response.getCallId().isEmpty());
```
Comment (ajrice6713) [inline src/test/java/com/bandwidth/MfaApiTests.java:58]: We want to assert status code and len of CallID here as well

**Review by hamermike — CHANGES_REQUESTED** [review]
@adamrbaldwin-bw could you please add some checks for the response status codes in these?

File: src/test/java/com/bandwidth/WebRtcApiTests.java [hunk 3]
```diff
@@ -21,19 +28,122 @@ public void initTest(){
...
 
-        participant = controller.createParticipant(ACCOUNT_ID, participant).getResult().getParticipant();
+        // Delete session
+        controller.deleteSession(ACCOUNT_ID, sessionCreationResponse.getId());
+        // expected response is an empty 204
 
-        controller.addParticipantToSession(ACCOUNT_ID, session.getId(), participant.getId(), null);
+        // Delete participant
+        controller.deleteParticipant(ACCOUNT_ID, participantCreationResponse.getParticipant().getId());
+        // expected response is an empty 204
```
Comment (ajrice6713) [inline src/test/java/com/bandwidth/WebRtcApiTests.java:147]: I think we can assert the status codes of the deletes here

**Review by ajrice6713 — CHANGES_REQUESTED** [review]

**Review by ajrice6713 — APPROVED** [review]

**Review by hamermike — APPROVED** [review]

**Pull request merged**